openassessment/xblock/staff_area_mixin.py

```diff
@@ -140,6 +140,9 @@ def get_staff_path_and_context(self):
                 self.get_staff_assessment_statistics_context(student_item["course_id"], student_item["item_id"])
             )
 
+        # Include whether or not this is a team assignment
+        context['is_team_assignment'] = self.teams_enabled
```

Comment:
There is a better flag in team_mixin called `is_team_assignment` that should be used here.